utils/textmate/Syntaxes/carbon.tmLanguage.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+  "name": "carbon",
+  "foldingStartMarker": "\\{\\s*$",
+  "foldingStopMarker": "^\\s*\\}",
+  "fileTypes": [
+    "carbon"
+  ],
+  "patterns": [
+    {
+      "include": "#comments"
+    },
+    {
+      "include": "#operators"
+    },
+    {
+      "name": "string.quoted.triple.carbon",
```

Comment:
We seem to define this twice now -- once here and again below, in `repository.strings`. Do we need both? Same for `string.quoted.double`. Can we remove the one below and keep this one?

Reply:
Yes, I have removed the strings pattern and kept the new one (from the plist)